### PR TITLE
Feature/andrewheyer/uml match xml add note to target v2

### PR DESCRIPTION
**Pull request by andrewbheyer**
When approved I will squash and merge.

### CODE REVIEW

File: python/lsst/sims/ocs/configuration/instrument/optics_loop_corr.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import lsst.pex.config as pexConfig
```
[the remark this answers is not in the document]
Comment (andrewbheyer) [reply python/lsst/sims/ocs/configuration/instrument/optics_loop_corr.py]: Configuration folder deleted.

**Comment by tribeiro** [comment]
Some places that needs fixes:

https://github.com/lsst-sims/sims_ocs/blob/23ab1f2a74ed680c81c1e9ac22740063afbf60a1/python/lsst/sims/ocs/kernel/simulator.py#L394

needs to be:

```python
self.cloud.bulkCloud = self.cloud_interface.get_cloud(self.time_handler.time_since_start)
```

https://github.com/lsst-sims/sims_ocs/blob/23ab1f2a74ed680c81c1e9ac22740063afbf60a1/python/lsst/sims/ocs/kernel/simulator.py#L396

needs to be:

```python
self.driver.update_external_conditions(self.cloud.bulkCloud, self.seeing.seeing)
```

**Comment by tribeiro** [comment]
This is a bug I introduced long ago but needs to be fixed here as well... 

on https://github.com/lsst-sims/sims_ocs/blob/23ab1f2a74ed680c81c1e9ac22740063afbf60a1/python/lsst/sims/ocs/kernel/sequencer.py#L126-L129

```python
            from SALPY_scheduler import scheduler_observationC
            from SALPY_scheduler import scheduler_observatoryStateC
            self.observation = scheduler_observationC()
            self.observatory_model.configure(obs_config)
            self.observatory_state = scheduler_observatoryStateC()
```

This will requires some changes on `ts_observatory_model` to work... I'll comment there...

**Comment by tribeiro** [comment]
Ok, now this is partially a bug that I introduced earlier. Not really a bug but it prevented your fix from automatically be applied. To fix it you first need to edit:

https://github.com/lsst-sims/sims_ocs/blob/23ab1f2a74ed680c81c1e9ac22740063afbf60a1/python/lsst/sims/ocs/kernel/sequencer.py#L126-L129

to be:

```python
            from SALPY_scheduler import scheduler_observationC
            from SALPY_scheduler import scheduler_observatoryStateC
            self.observation = scheduler_observationC()
            self.observatory_model.configure(obs_config)
            self.observatory_state = scheduler_observatoryStateC()
```

then on https://github.com/lsst-sims/sims_ocs/blob/23ab1f2a74ed680c81c1e9ac22740063afbf60a1/python/lsst/sims/ocs/kernel/simulator.py#L392-L393

you need to add:

```python 
                    self.driver.update_time(self.time_handler.current_timestamp, night)
                    driver_observatory_state = SALUtils.rtopic_observatory_state(observatory_state)
                    self.driver.update_internal_conditions(driver_observatory_state, night)
```

**Comment by tribeiro** [comment]
Ok, this is probably the final change needed.

https://github.com/lsst-sims/sims_ocs/blob/a512baae790db9de35d816e7684e993b4f39455c/python/lsst/sims/ocs/kernel/simulator.py#L541

needs to be:

```python
            filter_swap = self.driver.get_need_filter_swap()
            self.filter_swap.needSwap = filter_swap[0]
            self.filter_swap.filterToUnmount = filter_swap[1]
```

**Pull request merged**